README.md

```diff
@@ -7,3 +7,24 @@ This Action uses the [GitHub API](https://developer.github.com/v3/actions/workfl
 ```
 
 The cancellation might take a few seconds to process, this is most useful when you'd like to cancel a long running build. In combination with the `if` statement, this allows you to cancel the build conditionally.
+
+## Contributing
+
+We welcome contributions to improve the Cancel Action! To get started, please follow these guidelines:
```

Comment:
Erm, I'll honestly probably archive this action after you've made your pull request, so people can still use it but maintaining it isn't something I'm keen on doing 😄 I just wrote this to test that the API works more than anything.